scanner/cmd/scanner/main.go

```diff
@@ -72,30 +71,14 @@ func main() {
 
 	// Wait for signals.
 	sigC := make(chan os.Signal, 1)
-	signal.Notify(sigC, syscall.SIGINT, syscall.SIGTERM)
+	signal.Notify(sigC, os.Interrupt, unix.SIGTERM)
 	sig := <-sigC
-	zlog.Warn(ctx).Str("signal", sig.String()).Send()
+	zlog.Info(ctx).Str("signal", sig.String()).Send()
```

Comment:
`SIGTERM` is expected to happen in a pod's lifecycle, care to clarify why the change?

Reply:
if it's expected, then doesn't `info` make more sense than `warn`? `warn` sounds like it makes more sense for unexpected, yet ok, things